psydac/api/feec.py

```diff
@@ -86,51 +86,88 @@ def __init__(self, mapping, *spaces):
     #--------------------------------------------------------------------------
     @property
     def dim(self):
+        """dimension of the physical and logical space"""
         return self._dim
 
     @property
     def V0(self):
+        """First space of the De Rham sequence : H1 space"""
         return self._spaces[0]
 
     @property
     def V1(self):
+        """Second space of the De Rham sequence :
+        1d : L2 space
+        2d : either Hdiv or Hcurl space
+        3d : Hcurl space"""
         return self._spaces[1]
 
     @property
     def V2(self):
+        """Third space of the De Rham sequence :
+        2d : L2 space
+        3d : Hdiv space"""
         return self._spaces[2]
 
     @property
     def V3(self):
+        """Fourth space of the De Rham sequence : L2 space in 3d"""
         return self._spaces[3]
 
     @property
     def H1vec(self):
+        """Vectorial H1 space built has cartesian product of V0"""
         assert self.has_vec
         return self._H1vec
 
     @property
     def spaces(self):
+        """Spaces of the proper de Rham sequence (excluding Hvec)"""
         return self._spaces
 
     @property
     def mapping(self):
+        """The mapping from the logical space to the physical space of the discrete De Rham."""
         return self._mapping
 
     @property
     def callable_mapping(self):
+        """The mapping as a callable"""
         return self._callable_mapping
 
     @property
     def derivatives_as_matrices(self):
+        """Differential operators of the De Rham sequence as BlockLinearOperator"""
         return tuple(V.diff.matrix for V in self.spaces[:-1])
 
     @property
     def derivatives_as_operators(self):
+        """Differential operators of the De Rham sequence as DiffOperator"""
```

Comment:
`as DiffOperator objects`, and you can specify on which type of objects these operators act

Reply:
@vcarlier These operators act on `FemField` objects, i.e. they take as input a `FemField` belonging to their `domain` `FemSpace`, and return a `FemField` belonging to their `codomain` `FemSpace`.

Reply:
Is it really the place to put all this information? Maybe we should redirect to the documentation of DiffOperator (and create it)?

Reply:
Let's change the name of this function to `derivatives_as_fem_operators`. There is no need to mention class `DiffOperator` as this feels like an internal implementation detail to me.

Reply:
Actually maybe we could just call it `derivatives`, which is consistent with the `projectors` method.

Reply:
Called derivatives in [49ed53e](https://github.com/pyccel/psydac/pull/349/commits/49ed53eec0a2ac1e5117de0e818720027b2a6e36) 
 Looks like this method is anyway never used in the code at this point.

Reply:
you mean: in not a single branch 😅 ?

Reply:
I only scanned devel.
But the multipatch counterpart seems to be used in some multipatch tests.